src/npctrade.cpp

```diff
@@ -550,6 +555,7 @@ bool trading_window::perform_trade( npc &np, const std::string &deal )
         std::vector<item_pricing> &target_list = focus_them ? theirs : yours;
         size_t &offset = focus_them ? them_off : you_off;
         const std::string action = ctxt.handle_input();
+        const input_event evt = ctxt.get_raw_input();
```

Comment:
was it necessary to move this out? generally variables should be in the narrowest scope possible